sceptre/plan/actions.py

```diff
@@ -1000,3 +1000,115 @@ def diff(self, stack_differ):
         :rtype: sceptre.diffing.stack_differ.StackDiff
         """
         return stack_differ.diff(self)
+
+    @add_stack_hooks
+    def detect_stack_drift(self):
```

Comment:
The "stack" is implicit in this being a Stack actions. You could probably just call this `detect_drift`

Reply:
I am not sure I agree on this one point. The methods in this class are named after the Sceptre commands they implement. Changing the name of this one would break that convention.

Reply:
After refactoring this method is renamed `drift_show` which seems to resolve the original concern.